exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -105,8 +105,10 @@
   String HTTP_TRUSTSTORE_PATH = "javax.net.ssl.trustStore";
   String HTTP_TRUSTSTORE_PASSWORD = "javax.net.ssl.trustStorePassword";
   String SYS_STORE_PROVIDER_CLASS = "drill.exec.sys.store.provider.class";
+  String SYS_STORE_PROVIDER_ZK_BLOBROOT = "drill.exec.sys.store.provider.zk.blobroot";
   String SYS_STORE_PROVIDER_LOCAL_PATH = "drill.exec.sys.store.provider.local.path";
-  String SYS_STORE_PROVIDER_LOCAL_ENABLE_WRITE = "drill.exec.sys.store.provider.local.write";
+  String SYS_STORE_PROVIDER_EPHEMERAL_WRITE = "drill.exec.sys.store.provider.ephemeral.write";
+  String SYS_STORE_PROVIDER_EPHEMERAL_CAPACITY = "drill.exec.sys.store.provider.ephemeral.capacity";
```

Comment:
The Ephemeral store can also be used for non-profile related usage. Trying to keep it generic